frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx

```diff
@@ -17,6 +19,12 @@ export default class SettingsUpdatesForm extends Component {
   };
 
   renderVersionUpdateNotice() {
+    if (MetabaseSettings.isHosted()) {
+      return (
+        <div>{t`Metabase Cloud keeps your instance up to date. Thanks for being a customer!`}</div>
```

Comment:
Aware that introducing a new string atm is not ideal.

Please see discussion: https://github.com/metabase/metabase/issues/16792#issuecomment-875655045

cc @mazameli

Reply:
It's not a huge deal. Just let me know as soon as we merge this and I can get it translated. If we're going to do this one, incidentally, we may as well merge in that other typo PR you pinged me about, too.